src/uu/tr/src/operation.rs

```diff
@@ -319,13 +329,31 @@ impl Sequence {
             recognize(many_m_n(1, 3, one_of("01234567"))),
             |out: &[u8]| {
                 let str_to_parse = std::str::from_utf8(out).unwrap();
+                match u8::from_str_radix(str_to_parse, 8) {
+                    Ok(ue) => Some(ue),
+                    Err(_pa) => None,
+                }
+            },
+        )(input)
+    }
 
+    fn parse_octal_up_to_three_digits_with_warning(input: &[u8]) -> IResult<&[u8], u8> {
+        map_opt(
+            recognize(many_m_n(1, 3, one_of("01234567"))),
+            |out: &[u8]| {
+                let str_to_parse = std::str::from_utf8(out).unwrap();
                 match u8::from_str_radix(str_to_parse, 8) {
                     Ok(ue) => Some(ue),
                     Err(_pa) => {
```

Comment:
I think it is cleaner to simply use Err(`_`):
```suggestion
                    Err(_) => {
```